ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -230,6 +230,9 @@ RUN cd /home/user/.theia/ \
  && yarn \
  && yarn theia build
 
+# Install TypeScript globally to provide a language server fallback.
+RUN npm install -g typescript
```

Comment:
Interesting, what is this "language server fallback" mechanism you're referring to?

Also, if this a requirement to make Theia work, shouldn't we merge it with the Theia installation instructions? (No strong opinion on this, especially if a globally installed typescript can be useful to other projects.)

Reply:
Theia doesn't bundle TS due to the following reasons:
- it's optional
- different TS versions have minor incompatibilities

We will probably be adding things like RLS, clangd and other language servers, so I think it's a good idea to separate them.

Reply:
Cool, thanks!

> We will probably be adding things like RLS, clangd and other language servers

I believe we already pre-install [rls](https://github.com/JanitorTechnology/dockerfiles/blob/1a8dc2e04016994d2e6c76549a1ad99e37536de8/ubuntu-dev/ubuntu-dev.dockerfile#L170) and `clangd-6.0` (I guess it comes with [clang-6.0](https://github.com/JanitorTechnology/dockerfiles/blob/1a8dc2e04016994d2e6c76549a1ad99e37536de8/ubuntu-dev/ubuntu-dev.dockerfile#L29)).